Week03/lesson06/classwork/startup_matchmaker/Styles/styles.css

```diff
@@ -0,0 +1,182 @@
+.clearfix:before, .clearfix:after {
+  content: "";
+  display: table;
+}
+
+.clearfix:after{
+  clear: both;
+}
+
+*{
+box-sizing:border-box;
+}
+
+
+nav {
+  list-style-type: none;
+  float:right;
+  display:inline-block;
+  margin: 0.67em 0;
+}
+
+header{
+  background-color: black;
+}
+
+nav a {
+  color: white;
+  text-decoration: none;
+  font-size: 32px;
+}
+
+a:hover{
+  color:yellow;
+}
+
+
+li{
+  display:inline-block;
+  padding: 2.5px;
+}
+
+h1 {
+  color:white;
+  display:inline-block;
+}
+
+nav:last-child{
```

Comment:
My guess is that here, you were originally trying to add some `margin-right` between the elements inside of `<nav>`, that is, to the `<li>`'s. And then you wanted to make sure that the last item in the list didn't have that `margin-right`. You were SO SO close! You just added the `:last-child` to the wrong thing.

So the CSS should've been something along the lines of:

```
nav {
  float: right;
  display: inline-block;
  padding-right: 20px;
}

nav ul { /* you forgot to wrap the <li>'s in a <ul>, it's not semantically correct to have list items without a parent list */
  list-style-type: none;
}

nav li {
  display: inline-block;
  margin-right: 15px;
}

nav li:last-child {
  margin-right: 0;
}

nav a {
  padding: 15px; /* by adding the padding on the <a> we increase the hover area of the link to be larger than strictly over the text */
  display: block; /* this is what allows us to manipulate the padding/width/height of the usually inline <a> tag */
  color: white;
  text-decoration: none;
  font-size: 32px;
}
```